sdk/communication/communication-phone-numbers/review/communication-phone-numbers.api.md

```diff
@@ -133,6 +142,38 @@ export interface SearchAvailablePhoneNumbersRequest extends PhoneNumberSearchReq
     countryCode: string;
 }
 
+// @public
+export class SipRoutingClient {
+    constructor(connectionString: string, options?: SipRoutingClientOptions);
+    constructor(endpoint: string, credential: KeyCredential, options?: SipRoutingClientOptions);
+    constructor(endpoint: string, credential: TokenCredential, options?: SipRoutingClientOptions);
```

Comment:
If there is not specific documentation for each of the credential types it is recommended to use a union type instead of overload i.e.

```ts
constructor(endpoint: string, credential: TokenCredential | KeyCredential, options?: **SipRoutingClientOptions);**
```

Reply:
There is a specific documentation for each credential. 
The pattern was reused from the second client in the same SDK (PhoneNumbersClient).